.github/workflows/fork-ci.yaml

```diff
@@ -74,34 +74,42 @@ jobs:
 
   update_network_operator_values:
     needs:
-    - determine_docker_registry_and_tag
-    - build_and_push_images
+      - determine_docker_registry_and_tag
+      - build_and_push_images
     runs-on: ubuntu-latest
     env:
       DOCKER_TAG: ${{ needs.determine_docker_registry_and_tag.outputs.docker_tag }}
       GH_TOKEN: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}
     steps:
-    - uses: actions/checkout@v4
-      with:
-        token: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}
-        repository: ${{ github.repository_owner }}/network-operator
-    - name: Create PR to update image tags in network-operator values
-      run: |
-        git config user.name  nvidia-ci-cd
-        git config user.email svc-cloud-orch-gh@nvidia.com
-
-        git checkout -b feature/update-sriov-tags-to-$DOCKER_TAG
-        sed -Ei "s/(version: )network-operator-.+/\\1$DOCKER_TAG/g" hack/release.yaml
-        make release-build
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}
+          repository: ${{ github.repository_owner }}/network-operator
+      - name: Check if beta build
+        run: echo "IS_BETA=$(echo $DOCKER_TAG | awk -F  '-v' '/1/ {print $2}' | grep 'beta'" >> $GITHUB_ENV
+      - name: Get branch name
+        run: |
+          if [ -z "${IS_BETA}" ]; then
+            echo "BRANCH_NAME=master" >> $GITHUB_ENV
+          else
+            echo "BRANCH_NAME="$DOCKER_TAG" >> $GITHUB_ENV
+          fi
+      - name: Create PR to update image tags in network-operator values
+        run: |
+          git config user.name  nvidia-ci-cd
+          git config user.email svc-cloud-orch-gh@nvidia.com
+          
+          git checkout -b feature/update-sriov-tags-to-$DOCKER_TAG
+          sed -Ei "s/(version: )network-operator-.+/\\1$DOCKER_TAG/g" hack/release.yaml
+          make release-build
 
-        if ! git diff --color --unified=0 --exit-code; then
-          git commit -sam "task: update SR-IOV images tags to $DOCKER_TAG in chart values"
-          git push -u origin feature/update-sriov-tags-to-$DOCKER_TAG
-          gh pr create \
-            --repo ${{ github.repository_owner }}/network-operator \
-            --base master \
-            --head $(git branch --show-current) \
-            --fill \
-            --body "Created by the *${{ github.job }}* job in [${{ github.repository }} CI](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})."
-        fi
-      # TODO: double check with ivan about freddy's question: should we push to master or to a release branch?
+          if ! git diff --color --unified=0 --exit-code; then
+            git commit -sam "task: update SR-IOV images tags to $DOCKER_TAG in chart values"
+            git push -u origin feature/update-sriov-tags-to-$DOCKER_TAG
+            gh pr create \
+              --repo ${{ github.repository_owner }}/network-operator \
+              --base $BRANCH_NAME \
```

Comment:
and respective of my previous comment, a more suiting variable name here would be:
```bash
--base $BASE_BRANCH \
```